packages/camera/camera_avfoundation/CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## NEXT
 
 * Updates minimum supported SDK version to Flutter 3.19/Dart 3.3.
+* Adds lensType in the PlatformCameraDescription
```

Comment:
We have to bump the version in order to publish the change.

Reply:
bumped to 3.22/Dart 3.4